tests/pipeline/resources/no-topics/pipeline.yaml

```diff
@@ -0,0 +1,13 @@
+- type: producer
```

Comment:
isn't that the same file as `tests/pipeline/resources/custom-config/pipeline.yaml`?

Reply:
indeed they are.. i can get rid of one of them